content/2017/1/25/parallax.rst

```diff
@@ -0,0 +1,36 @@
+public: yes
+author: sondra
+tags: [design]
+image:
+  - src: '2017/parallax/parallax.jpg'
+summary: |
+  Whether you like it not, parallax scrolling is a reality, so let's learn how to do it well with CSS 3D. Paul Lewis' guide to performant parallaxing is the perfect place to start.
+
+
+Let's Talk About Parallax: Dos and Don'ts
+=========================================
+
+First, I'd like to thank `Rachel Nabors`_, award winning cartoonist and digital storyteller, for pointing us to Paul Lewis' excellent `article on performant parallaxing`_.
+
+.. _Rachel Nabors: http://rachelnabors.com/
+.. _article on performant parallaxing: https://developers.google.com/web/updates/2016/12/performant-parallaxing?utm_content=buffer29a95&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer
+
+Don'ts
+------
+
+Paul, a developer at Google, starts with the don'ts. Don't use scroll events to create a parallax effect.
+
+*"JavaScript doesn’t guarantee that parallaxing will keep in step with the page’s scroll position."*
```

Comment:
I thought from your sample that the blockquote syntax was a little arrow, but that doesn't appear to have worked.